docs/lang/articles/basic/field.md

```diff
@@ -133,6 +133,15 @@ the first is for field indexing, and the second is for matrix indexing.
 - The 0-D matrix field `x = ti.Matrix.field(n=3, m=4, dtype=ti.f32, shape=())` should be accessed by `x[None][p, q]` (`0 <= p < n, 0 <= q < m`).
 - `ti.Vector` is simply an alias of `ti.Matrix`.
 
+### Example
+This example helps you understand element and member in matrix fields:
+``` python
+X = ti.Matrix.field(n = 2, m = 3, dtype = ti.f32, shape = (2, 2))
+Element = X[0, 0]
+Member = X[0, 1][1,1]
+```
+![matrix_field](./matrix_field.jpg)
```

Comment:
The pictures of doc website should push to [here](https://github.com/taichi-dev/public_files/tree/master/taichi/doc) and use url link like `https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/doc/sparse_grids_2d.png`